package.json

```diff
@@ -10,7 +10,8 @@
     "test-wd": "mochify --wd test/client/**/*.spec.js",
     "test-cover": "mochify --cover test/client/**/*.spec.js",
     "test": "npm run phantom && npm run wd && npm run cover",
-    "test-server": "nodemon --watch test/server/**/*.js --watch sync-server.js --watch lib/sync-server.js test/server/test-server.js | bunyan"
+    "test-server": "nodemon --watch test/server/**/*.js --watch sync-server.js --watch lib/sync-server.js test/server/test-server.js | bunyan",
+    "eslint": "grunt"
```

Comment:
I've revisited this now and added an `npn run-script grunt-eslint` target to package.json. The reason for doing this is that `npm test` fails (which I think is because it requires some additional set up of PhantomJS but I'm not sure but that seems outside of the scope of this task). Having this run-script task allows us to run eslint by Travis.
@nialldonnellyfh Can you take a look and see what you think.